core/plugins/org.polarsys.capella.core.ui.search/src/org/polarsys/capella/core/ui/search/CapellaSearchResult.java

```diff
@@ -32,7 +38,22 @@
   public CapellaSearchResult(CapellaSearchQuery capellaSearchQuery) {
     this.capellaSearchQuery = capellaSearchQuery;
     setActiveMatchFilters(new MatchFilter[] {});
-    treeData = new TreeData(new ArrayList<Object>(), null);
+    treeData = new TreeData(new ArrayList<Object>(), null) {
+      // Customize this to add notes into search scope
+      @Override
+      protected Object doGetParent(Object element) {
+        if (element instanceof DRepresentationDescriptor) {
+          return ((DRepresentationDescriptor) element).getTarget();
+        } else if (element instanceof Shape) {
+          Diagram diagram = ((Shape) element).getDiagram();

```

Comment:
same here

Reply:
This works because Shape objects can only be found once their diagram is already loaded.

Reply:
:+1: